### PR TITLE
feat: add images assets support

### LINKED ISSUE #229: Add support for images inside designs
**Is your feature request related to a problem? Please describe.**
Images used as assets inside designs aren't currently supported. We should fix that.

Sketch file used to test this support: [xlayer.test.sketch.zip](https://github.com/xlayers/xlayers/files/2928357/xlayer.test.sketch.zip)

### PR DESCRIPTION
**Pull request by manekinekko**
Fixes #229

### CODE REVIEW

File: packages/xlayers/src/app/editor/viewer/lib/sketch.service.spec.ts [hunk 1]
```diff
@@ -21,15 +20,17 @@ describe('SketchService', () => {
     sketchService = TestBed.get(SketchService);
   });
 
-  it('should convert sketch to json', (done: DoneFn) => {
+  fit('should convert sketch to json', async(() => {
```
Comment (manekinekko) [inline packages/xlayers/src/app/editor/viewer/lib/sketch.service.spec.ts]: remove this `fit`

File: packages/xlayers/src/app/editor/code-editor/editor-container/codegen/shared-codegen.service.ts [hunk 2]
```diff
@@ -106,15 +109,41 @@ export class SharedCodegen {
...
+        innerContent.push(this.closeTag('span'));
+      } else if ((ast as any)._class === 'bitmap') {
+        let base64Content = this.sketchService.getImageDataFromRef((ast as any).image._ref).source;
+        base64Content = base64Content.replace('data:image/png;base64', '');
+
+        const attributes = [
+          `${classNameAttr}="${(ast as any).css__className}"`,
+          `role="${ast._class}"`,
+          `aria-label="${ast.name}"`,
+          `src="${this.buildImageSrc(
```
Comment (manekinekko) [inline packages/xlayers/src/app/editor/code-editor/editor-container/codegen/shared-codegen.service.ts]: consider reverting to the previous solution: using raw base64 image data.

File: packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts [hunk 3]
```diff
@@ -78,10 +78,11 @@ export class SketchContainerComponent implements OnInit {
     try {
       const data = await this.service.process(file);
       this.store.dispatch(new CurrentFile(data));
-    } catch (e) {
+    } catch (error) {
+      console.error(error);
```
Comment (Jefiozie) [inline packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts]: I would remove this unless there is a specific reason for it?

File: packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts [hunk 4]
```diff
@@ -78,10 +78,11 @@ export class SketchContainerComponent implements OnInit {
...
       this.store.dispatch(
         new InformUser(
-          'The design was created using an unsupported version of SketchApp.'
+          `Runtime Error: ${error}`
```
Comment (Jefiozie) [inline packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts]: I agree that when this happens it should not be a message as previous.
But maybe we can be a bit more helpful when this occurs. Something like: `If this already happend 
 multiple times please help us and make a issue`

What do you think?

**Comment by Jefiozie** [comment]
Tested it everything looks good 👍. Two really small comments but approved!

File: packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts (hunk 4 above)
Comment (manekinekko) [reply packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts]: Yep. We can append a report link thank points to https://github.com/xlayers/xlayers/issues/new?assignees=&labels=&template=bug_report.md&title=

File: packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts (hunk 3 above)
Comment (manekinekko) [reply packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts]: The reason is that if a runtime error occurs, we wont see it. If we ask sone user for an error message in their console, they wont see anything.

**Pull request merged**